tests/integration_tests/__init__.py

```diff
@@ -79,8 +72,38 @@ def setUp(self):
         self.addCleanup(self.driver.quit)
 
         self.driver.root_uri = self.get_server_url()
-
-        super(TestUI, self).setUp()
+        self.driver.implicitly_wait(30)
+        self.verificationErrors = []
+        self.wait = WebDriverWait(self.driver, 60)
+        self.accept_next_alert = True
+
+        super(IntegrationTestCase, self).setUp()
+
+    def is_element_present(self, how, what):
```

Comment:
These functions are auto generated by Selenium IDE and will be used during more complicated tests. Checking them in now do we don't have to later.

Reply:
Can you add some comments to that effect (ie "functions exported from Selenium IDE plugin")?